helm/values-dev.yaml

```
@@ -132,6 +133,8 @@ microservice-chart:
     PM_ORACLE_PASSWORD: pm-oracle-db-password
     PM_ORACLE_HOST: pm-oracle-db-host
     MONGO_PASSWORD: mongo-ecommerce-password
+    SECURITY_API_KEY_PRIMARY: helpdesk-service-requests-primary-api-key
```

```suggestion
    SECURITY_API_KEY_PRIMARY: ecommerce-helpdesk-service-primary-api-key
```

this is the secret name as defined into infra [here](https://github.com/pagopa/pagopa-infra/blob/fa9798d17fbec8e36d9ca98475b42d6472badef1/src/domains/ecommerce-common/02_security.tf#L866)